README.md

```diff
@@ -1 +1,67 @@
-# nmpolicy
\ No newline at end of file
+---
+---
+
+# Introduction
+
+An expressions driven declarative API for dynamic network configuration
+
+# Motivation
```

Comment:
Please add to the motivation section an example of how we updated multiple nodes in the past vs with nmpolicy.

Reply:
Will just add the past version since new version is on the presented example.